FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -0,0 +1,418 @@
+#!/bin/bash
```

Comment:
I see that we built longitudinal options into `PostFreeSurferPipeline`.  Did we decide it wasn't practical to do the same with `FreeSurferPipeline` (rather than add an stand-along "LongitudinalFreeSurferPipeline` script)?

Reply:
I don't know if you did before I joined the development. I picked on existing LongitudinalFreesruferPipeline.
After reviewing the original FreesurferPipeline, in my view merging longitudinal and cross-sectional options isn't obviously beneficial as longitudinally I can't reuse most of the cross-sectional code as I do with PostFreesurfer. Also, cross-sectional FS must be run before longitudinal FS on all timepoints so it also makes sense to have them in separate scripts.

Reply:
Jure/Grega implemented the LongitudinalFreeSurferPipeline.